pkg/http-server/routes.go

```diff
@@ -37,7 +37,7 @@ func (g *APIServer) Routes(configFile string) []*Route {
 		{verb: "POST", path: versionedPath("/{iac}/{iacVersion}/{cloud}/remote/dir/scan"), fn: h.scanRemoteRepo},
 
 		// k8s webhook Routes
-		{verb: "GET", path: "/k8s/webhooks/logs", fn: h.getLogs},
+		{verb: "GET", path: "/k8s/webhooks/{apiKey}/logs", fn: h.getLogs},
```

Comment:
missed this earlier, but, I think having `{apiKey}` in the url is not a good idea, as it would be exposed in the url!

should this auth filter be also applied to `/k8s/webhooks/logs/{uid}` route as well?

Reply:
can be taken up later